api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -422,6 +456,9 @@ message Plan {
   // Metadata for telemetry received during the pass. Only populated when the pass has completed
   // successfully and data processing is complete.
   repeated TelemetryMetadata telemetry_metadata = 13;
+
+  // The channel set used to reserve this plan.
+  ChannelSet channel_set = 16;
```

Comment:
It's a bit of a pain to manage, but we could mark this as 'alpha' and remove the deprecation from the other token until everything is live in prod.